src/inputs/durationPicker.jsx

```diff
@@ -7,38 +7,59 @@ import Input from './input';
 
 const propTypes = {
     className: PropTypes.string,
+    /**
+     * A TableRow can be disabled.
```

Comment:
Oops ... copy-and-paste error.  Should be "A DurationPicker can be disabled."

Reply:
Fixing...

Reply:
Commit 7c8b691 fixes this.

Reply:
Looks good.  Resolving.